plugins/org.jkiss.dbeaver.ext.altibase.ui/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,34 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: %Bundle-Name
+Bundle-SymbolicName: org.jkiss.dbeaver.ext.altibase.ui;singleton:=true
+Bundle-Version: 1.0.0.qualifier
+Bundle-Release-Date: 20220627

```

Comment:
You can use the current date here.

Reply:
Okay, I changed the release date two MANIFEST.MF files at org.jkiss.dbeaver.ext.altibase.ui and org.jkiss.dbeaver.ext.altibase files.